### ISSUE TITLE
Add `Import library with prefix` option

### ISSUE THREAD

**Issue by FMorschel** [Dart-Code/Dart-Code#5123]
**Is your feature request related to a problem? Please describe.**

In my company, we have some packages for internal use and in the past there have been times where a package class was extended/implemented with the same name inside the package. This gives room to conflicts when both the package and the class are imported in another file. 

Other use-cases would be `dart:math`, `dart:ui` and `package:http/http.dart` which are almost always imported with prefixes.

**Describe the solution you'd like**
I'd like to have an option of an `import library` quick-fix that asks for a prefix or moves the cursor after the `as` and before the `;` on the import line.

**Describe alternatives you've considered**
Manually looking for the import and adding the prefix.

**Additional context**
It would be _perfect_ if this could parse the prefix if prewritten on the method/class/etc like:

```dart
dev.log //<- this would ask to import either `dart:math` or `dart:developer` with the prefix `dev`.
```

**Comment by DanTup** [comment]
This sounds like an interesting idea - I've often written `path.x` or `math.x` to find it wasn't imported and then ended up adding it without the prefix to fix up after.

Would you mind filing the request at https://github.com/dart-lang/sdk and posting a link back here? These code actions come from the Dart analysis server (which calls them "assists") so this would need to be implemented there.

Thanks!

**Comment by FMorschel** [comment]
> Would you mind filing the request at https://github.com/dart-lang/sdk

Done!

> [...] and posting a link back here?

I'm not sure what you meant. Is this mention enough?

**Comment by DanTup** [comment]
Yes, thanks! It was just in case others find this issue looking for the same thing, they can jump to the SDK issue to add 👍/comments :-)
